elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/DynamicConfigTest.java

```diff
@@ -1,113 +0,0 @@
-/*
- * Copyright 2020, Yahoo Inc.
- * Licensed under the Apache License, Version 2.0
- * See LICENSE file in project root for terms.
- */
-package example.tests;
-
-import static com.yahoo.elide.test.jsonapi.JsonApiDSL.attr;
-import static com.yahoo.elide.test.jsonapi.JsonApiDSL.attributes;
-import static com.yahoo.elide.test.jsonapi.JsonApiDSL.data;
-import static com.yahoo.elide.test.jsonapi.JsonApiDSL.id;
-import static com.yahoo.elide.test.jsonapi.JsonApiDSL.resource;
-import static com.yahoo.elide.test.jsonapi.JsonApiDSL.type;
-import static io.restassured.RestAssured.when;
-import static org.hamcrest.CoreMatchers.hasItems;
-import static org.hamcrest.Matchers.equalTo;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import com.yahoo.elide.core.exceptions.HttpStatus;
-import org.junit.jupiter.api.Test;
-import org.springframework.test.context.jdbc.Sql;
-import org.springframework.test.context.jdbc.SqlMergeMode;
-
-/**
- * Dynamic Configuration functional test.
- * TODO - All of the tests in this file need to migrate over to AggregationDataStore so they are not duplicated
- * here and in standalone tests.
- */
-@SqlMergeMode(SqlMergeMode.MergeMode.MERGE)
-@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
-        statements = "CREATE TABLE PlayerStats (name varchar(255) not null,"
-                + "\t\t countryId varchar(255), createdOn timestamp, updatedOn timestamp,"
-                + "\t\t highScore bigint, primary key (name));"
-                + "CREATE TABLE PlayerCountry (id varchar(255) not null,"
-                + "\t\t isoCode varchar(255), primary key (id));"
-                + "INSERT INTO PlayerStats (name,countryId,createdOn,updatedOn) VALUES\n"
-                + "\t\t('SerenaWilliams','1','2000-10-10','2001-10-10');"
-                + "INSERT INTO PlayerCountry (id,isoCode) VALUES\n"
-                + "\t\t('2','IND');"
-                + "INSERT INTO PlayerCountry (id,isoCode) VALUES\n"
-                + "\t\t('1','USA');")
-@Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD,
-statements = "DROP TABLE PlayerStats; DROP TABLE PlayerCountry;")
-public class DynamicConfigTest extends IntegrationTest {
-    /**
-     * This test demonstrates an example test using the JSON-API DSL.
-     * @throws InterruptedException
-     */
-
-    @Test
-    public void jsonApiGetTest() {
```

Comment:
We can move the filter test cases to Aggregation store IT test. But I think there may be some merit in running `DynamicConfigTest.jsonApiGetTest()` in Spring separately. It will help us in testing and making sure that any changes to bean configurations do not break Spring Applications. 

@aklish Thoughts?

Reply:
Instead of a separate java file, may be it could just run as part of ControllerTest or AggregationStoreTest